projects/1.16/assets/draconic-evolution/draconicevolution/lang/zh_cn.json

```
@@ -44,240 +48,245 @@
-  "gui.draconicevolution.draconium_chest.autofill_all": "Autofill All",
-  "gui.draconicevolution.draconium_chest.autofill_all.info": "Autofill All",
-  "gui.draconicevolution.draconium_chest.autofill_fill": "Autofill Fill",
-  "gui.draconicevolution.draconium_chest.autofill_fill.info": "Autofill Fill",
-  "gui.draconicevolution.draconium_chest.autofill_lock": "Autofill Lock",
-  "gui.draconicevolution.draconium_chest.autofill_lock.info": "Autofill Lock",
-  "gui.draconicevolution.draconium_chest.autofill_off": "Autofill Off",
-  "gui.draconicevolution.draconium_chest.autofill_off.info": "Autofill Off",
-  "gui.draconicevolution.draconium_chest.color_picker": "更改颜色",
-  "gui.draconicevolution.draconium_chest.color_picker.info": "更改颜色",
-  "gui.draconicevolution.draconium_chest.smelt_energy": "Smelt Energy",
-  "gui.draconicevolution.draconium_chest.smelt_speed": "Smelt Speed",
-  "gui.draconicevolution.energy_net.bound_to_invalid": "工具被绑定到无效的方块上！",
+  "gui.draconicevolution.dislocator.double_click_name": "双击左键重命名。",
+  "gui.draconicevolution.dislocator.drag_to_move": "左键点击并拖拽移动。",
+  "gui.draconicevolution.dislocator.edit_lock.info": "切换编辑锁定",
+  "gui.draconicevolution.dislocator.fuel": "传送燃料：",
+  "gui.draconicevolution.dislocator.fuel.info": "剩余的传送总数。",
+  "gui.draconicevolution.dislocator.fuel_add_1.info": "从你的物品栏中提取1个末影珍珠。\n末影珍珠被用作传送燃料。",
+  "gui.draconicevolution.dislocator.fuel_add_16.info": "从你的物品栏中提取16个末影珍珠。\n末影珍珠被用作传送燃料。",
+  "gui.draconicevolution.dislocator.fuel_add_all.info": "从你的物品栏中提取所有末影珍珠。\n末影珍珠被用作传送燃料。",
+  "gui.draconicevolution.dislocator.mode_blink": "使用方式：\n瞬移",
+  "gui.draconicevolution.dislocator.mode_tp": "使用方式：\n传送",
+  "gui.draconicevolution.dislocator.must_unlock": "（必须关闭编辑锁定）",
+  "gui.draconicevolution.dislocator.right_click_mode.info": "右键点击，在“传送至选定位置”和“瞬移”之间切换\n你还可以配置对应的键位。",
+  "gui.draconicevolution.dislocator.right_click_tp": "右键点击以传送",
+  "gui.draconicevolution.dislocator.update": "将位置设置在此",
+  "gui.draconicevolution.dislocator.update.info": "将所选位置更新为你当前的坐标。",
+  "gui.draconicevolution.draconium_chest.color_picker": "改变龙箱颜色",
+  "gui.draconicevolution.draconium_chest.feed.all.info": "自动烧制：全部\n将所有可以烧制的物品放入熔炉。",
+  "gui.draconicevolution.draconium_chest.feed.filter.info": "自动烧制：过滤\n将所有可烧制的物品放入熔炉。",
+  "gui.draconicevolution.draconium_chest.feed.off.info": "自动烧制：关",
+  "gui.draconicevolution.draconium_chest.feed.sticky.info": "自动烧制：过滤（粘性）\n与“自动烧制：过滤”基本相同，但是最终会在熔炉内保留一个物品不烧炼，以维持过滤项。",
+  "gui.draconicevolution.energy_net.bound_to_invalid": "该工具绑定的是一个无效方块！",
   "gui.draconicevolution.energy_net.clear_links": "清除链接",
   "gui.draconicevolution.energy_net.clear_receivers": "清除接收器",
-  "gui.draconicevolution.energy_net.device_invalid": "无效设备！",
-  "gui.draconicevolution.energy_net.devices_linked": "设备已连接！",
+  "gui.draconicevolution.energy_net.device_invalid": "那不是一个有效的设备！",
+  "gui.draconicevolution.energy_net.devices_linked": "设备已链接！",
```

```suggestion
  "gui.draconicevolution.energy_net.devices_linked": "设备已连接！",
```
嗯……虽然的确是改回去了，但我查到的情况是“链接”这个词更偏向计算机网络方面的数据链接，而非其他方面的连接，或许这里用“连接”更恰当一点？